index.bs

```diff
@@ -215,8 +220,9 @@ spec:url; type:dfn; text:scheme
     portals-rendering.html
   </wpt>
 
-  A <{portal}> element may have a <dfn for="HTMLPortalElement">guest browsing context</dfn>, which is a
-  [=portal browsing context=]. If so, the element is the [=host=] of its [=guest browsing context=].
+  A <{portal}> element has a <dfn for="HTMLPortalElement">guest browsing context</dfn>, which is
```

Comment:
This is hard to read. Can we simplify it somehow?

Reply:
Introduced a variable to replace the pronoun "it". English is so terrible for when I really want to say "GuestBrowsingContext = HostElement<sup>-1</sup>".